lib/tasks/feed_management.rake

```diff
@@ -36,20 +36,29 @@ namespace :feeds do
   task :retrieve_items, [:feed_slug] => :environment do |task, args|
     feeds = nil
     feed_slug = args[:feed_slug]
-    feeds = feed_slug.present? ? FeedSource.where(slug: feed_slug) : FeedSource.all
+    feeds = feed_slug.present? ? FeedSource.active.where(slug: feed_slug) : FeedSource.active
 
     batch_process(feeds) do |feed|
       puts feed.name
       feed.retrieve!
     end
   end
 
+  #FIXME This isn't a reliable way to parse the twitter_user because some of them
+  #have twitter urls like "https://twitter.com/cardanostiftung?lang=en"
+  #not going to fix it now because not sure if we'll ever use this again
   desc "Build Feeds Source for all coins twitter feeds"
   task :build_all_feed_sources_for_twitter  => :environment do
     #using batch_process here just for error_handling e.g. since we have manually created one of these already
     #we just want it to skip the unique constraint error that will be thrown and keep going to create the rest
     batch_process(Coin.where.not(twitter: nil).where.not(twitter: "")) do |coin|
-      FeedSource.create_from_coins_twitter!(coin)
+    twitter_user = coin.twitter.split("/").last
```

Comment:
Needs tabbing for proper alignment please. 👾

Reply:
Just to note, this was caught by CodeClimate, so please do pay attention to the build!

Reply:
@hanchang sorry .... not seeing what you were referring to here.

Reply:
@hanchang Ah... I see it now.  Thanks.